File: 2010/blokjes/main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var amount int
	fmt.Scanf("%d\n", &amount)

	for i := 1; i <= amount; i++ {
		var n int
		fmt.Scanf("%d\n", &n)
		sum := 0
		for c := 1; c < n+1; c++ {
			sum += int(math.Pow(float64(c), 3.0))
		}
		fmt.Println(sum)
	}

}

File: 2010/blokjes/main.rs
use std::io;

fn read_unsigned_integer() -> u32 {
    let mut integer = String::new();
    io::stdin().read_line(&mut integer).unwrap();
    integer.trim().parse().unwrap()
}

fn main() {
    let amount = read_unsigned_integer();
    for _i in 0..amount {
        let n = read_unsigned_integer();

        let mut sum = 0;
        
        for c in 1..(n+1) {
            sum += c.pow(3);
        }
        
        println!("{}", sum);
    }
}